src/test/java/com/iota/iri/TransactionTestUtils.java

```diff
@@ -34,4 +36,24 @@ public static TransactionViewModel createTransactionWithTrunkBundleHash(Transact
                 TransactionViewModel.BUNDLE_TRINARY_OFFSET, TransactionViewModel.BUNDLE_TRINARY_SIZE);
         return tx;
     }
+
+    public static TransactionViewModel createTransactionWithTrytes(String trytes) {
+        String expandedTrytes  = expandTrytes(trytes);
+        byte[] trits = Converter.allocatingTritsFromTrytes(expandedTrytes);
+        return new TransactionViewModel(trits, Hash.calculate(SpongeFactory.Mode.CURLP81, trits));
```

Comment:
shouldn't we just add a default constructor to TxVM or a helper method somewhere?

Reply:
It has this `expandTrytes` call.
I don't want this as part of default behavior...